src/main.jsx

```diff
@@ -0,0 +1,19 @@
+import reactShim from "./react-shim";
+import React from "react";
+import { render } from "react-dom";
+import App from "./EosForm.jsx";
+
+function main() {
+  let dialog;
+  function getDialog() {
+    if (dialog == null) {
+      dialog = document.createElement("dialog");
+      render(<App dialog={dialog} />, dialog);
+    }
+    return dialog;
+  }
+  return document.body.appendChild(getDialog()).showModal();
+}
+export const commands = {
```

Comment:
it would be cleaner and more scalable to move the declaration outside of the export